frontend/src/metabase/querying/utils/filters/utils.ts

```diff
@@ -33,3 +33,10 @@ export function getDefaultAvailableOperator<T extends Lib.FilterOperatorName>(
     options[0].operator
   );
 }
+
+/**
+ * Returns indexes of stages from which columns are exposed for filtering
+ */
+export function getFilterStageIndexes(query: Lib.Query): number[] {
```

Comment:
Should we be reusing the same function from `frontend/src/metabase-lib/v1/parameters/utils/targets.ts`?

Reply:
05ec9b145b1bb47c30950f66881181cf5c30093e